Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -180,15 +172,6 @@ const Contest = () => {
         [ changeCurrentHash ],
     );
 
-    useEffect(
-        () => () => {
```

Comment:
What is the solution to the problem? If you only remove it, it creates 2 queries for GetSubmissionResultsByProblem.

Reply:
Another solution should be implemented, as this introduces more problems than it solves.
I am reopening the issue: https://github.com/SoftUni-Internal/exam-systems-issues/issues/774
@PetarNeshkov if you have suggestions or ideas, you can share them.